docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -45,10 +45,10 @@ To use Iceberg, you need:
 Configuration
 -------------
 
-The connector supports two Iceberg catalog types, you may use either a Hive
-metastore service (HMS) or AWS Glue. The catalog type is determined by the
+The connector supports three Iceberg catalog types, you may use either a Hive
```

Comment:
Let's make it more abstract because probably other catalog types will eventually follow.

`several` ?